extensions/drafts/src/quick-capture.tsx

```diff
@@ -0,0 +1,19 @@
+import { LaunchProps, showHUD } from "@raycast/api";
+import { runAppleScript } from "@raycast/utils";
+
+export default async function main(props: LaunchProps<{ arguments: Arguments.QuickCapture }>) {
+  const { content } = props.arguments;
+
+  const res = await runAppleScript(
+    `
+on run argv
+  tell application "Drafts"
+    make new draft with properties {content:argv, flagged:false}
+  end tell
+end run
+  `,
+    [content]
+  );
+
+  await showHUD(res);
```

Comment:
The result shows "draft [uuid]" which is not really helpful for the user.

I suggest to change this to
```suggestion
  await showHUD("Created Draft 👍", { clearRootSearch: true, popToRootType: PopToRootType.Immediate });
```

similar to the 'create-draft-background.tsx'